segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/PravegaRequestProcessor.java

```diff
@@ -580,6 +620,15 @@ public void mergeSegmentsBatch(WireCommands.MergeSegmentsBatch mergeSegments) {
                });
     }
 
+    private CompletableFuture<Void> appendOnIndexSegment(String segmentName) {
```

Comment:
Instead of **CompletableFuture<Void>** can we make this method return type as void?